src/fu-tool.c

```diff
@@ -1334,20 +1334,21 @@ fu_util_activate (FuUtilPrivate *priv, gchar **values, GError **error)
 {
 	gboolean has_pending = FALSE;
 	g_autoptr(FuHistory) history = fu_history_new ();
-	g_autoptr(GPtrArray) devices = NULL;
+	g_autoptr(GPtrArray) history_devices = NULL;
+	g_autoptr(GPtrArray) engine_devices = NULL;
```

Comment:
maybe `s/history_devices/devices` and `s/engine_devices/devices_engine`?

Reply:
fixed